builtin/providers/ultradns/config.go

```diff
@@ -22,7 +22,7 @@ func (c *Config) Client() (*udnssdk.Client, error) {
 		return nil, fmt.Errorf("Error setting up client: %s", err)
 	}
 
-	log.Printf("[INFO] UltraDNS Client configured for user: %s", client.Username)
+	log.Printf("[INFO] UltraDNS Client configured for user: %s", c.Username)
```

Comment:
I can't see anywhere where `client` is renamed to `c`, is this change still good? I'm probably missing something.